### PR TITLE
Make prettier JS snippets

### PR DESCRIPTION
**Pull request by brikou**

### CODE REVIEW

**Comment by brikou** [comment]
I signed it!

File: docs/api.md [hunk 1]
```diff
@@ -429,10 +432,11 @@ puppeteer.launch().then(async browser => {
   await page.exposeFunction('readfile', async filePath => {
     return new Promise((resolve, reject) => {
       fs.readFile(filePath, 'utf8', (err, text) => {
-        if (err)
+        if (err) {
```
Comment (Garbee) [inline docs/api.md]: This ins't the code style for the project. When if blocks have one statement executions the curlies are ignored.

Comment (brikou) [reply docs/api.md]: Ok I'll revert then

File: docs/api.md [hunk 2]
```diff
@@ -633,7 +635,10 @@ puppeteer.launch().then(async browser => {
   let page = await browser.newPage();
   await page.setRequestInterceptionEnabled(true);
   page.on('request', interceptedRequest => {
-    if (interceptedRequest.url.endsWith('.png') || interceptedRequest.url.endsWith('.jpg'))
+    if (
```
Comment (ebidel) [inline docs/api.md]: I think this is more legible on one line.

File: docs/api.md [hunk 3]
```diff
@@ -761,8 +766,14 @@ const puppeteer = require('puppeteer');
 puppeteer.launch().then(async browser => {
   let page = await browser.newPage();
   let currentURL;
-  page.waitForSelector('img').then(() => console.log('First URL with image: ' + currentURL));
-  for (currentURL of ['https://example.com', 'https://google.com', 'https://bbc.com'])
+  page
+    .waitForSelector('img')
+    .then(() => console.log('First URL with image: ' + currentURL));
+  for (currentURL of [
```
Comment (ebidel) [inline docs/api.md]: Keep single line

File: docs/api.md [hunk 4]
```diff
@@ -1065,8 +1076,14 @@ const puppeteer = require('puppeteer');
...
```
Comment (ebidel) [inline docs/api.md]: keep single line.

**Review by ebidel — CHANGES_REQUESTED** [review]

**Review by ebidel — APPROVED** [review]

**Pull request merged**